package.json

```diff
@@ -14,11 +14,12 @@
   "homepage": "https://github.com/alphagov/govuk-design-system#readme",
   "scripts": {
     "preinstall": "node bin/check-nvmrc.js",
-    "prestart": "node bin/check-nvmrc.js",
+    "prestart": "node bin/check-nvmrc.js && npm run test",
```

Comment:
Not sure about usefulness of this – it'll only run once, often before you've actually made any changes, and it adds a good 10-15 seconds of linting and testing before the server actually starts. I'd be happy enough with tests just being run by Travis and on-demand locally, personally.

Reply:
Fair, running it with CI should be enough.